src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = days * 40;
+
+  if (days >= 7) {
+    return cost - 50;
+  } else if (days >= 3) {
+    return cost - 20;
+  } else {
+    return cost;
```

Comment:
Remove the 'else' keyword after the 'if' blocks that contain a 'return' statement to simplify the code structure.

Reply:
Still not fixed